### PR TITLE
log() first PR

### PR DESCRIPTION
**Pull request by Dazeofcitywalk**
Summary:
finish the log() function

PR details:
· import data frame from csv file
· add a new line on the top of data frame
· export data frame to csv file

### CODE REVIEW

File: Calendar_CLI/Log.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# File log
```
Comment (YLee2121) [inline Calendar_CLI/Log.py]: remove this line

Comment (Dazeofcitywalk) [reply Calendar_CLI/Log.py]: removed

File: Calendar_CLI/Log.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import datetime
+import pandas as pd
+
+def write_log(username: str, function_name: str) -> None():
+    """
+    Track the datetime, user's name, and activities.
+    All the logs should be written in inverse chronological order.
```
Comment (YLee2121) [inline Calendar_CLI/Log.py:16]: image

Please follow the documentation guideline

**Review by YLee2121 — CHANGES_REQUESTED** [review]
Only a little bit needs to change.

File: Calendar_CLI/Log.py (hunk 2 above)
Comment (Dazeofcitywalk) [reply Calendar_CLI/Log.py:16]: Thanks for reminding.
Modified (but choose the function template, instead of method(class) template above.

Comment (YLee2121) [reply Calendar_CLI/Log.py:16]: No problem! 

As long as the documentation shows the description and and parameter, it's all good!

**Review by YLee2121 — APPROVED** [review]
## **Approve. Please resolve the conflict before I can merge.**

@Dazeofcitywalk there is a conflict for you to solve.

---

Here is some feedback for the future. 

image

In the commit message, the lable is not simply acting as a verb of what you do. Please rmb to add the lable, and also please try to refer the commit guideline. 

For example, in your commit, you use `write`; however, it's better to use `feat`, which means feature.

**Pull request merged**